xray/services/buildscan.go

```diff
@@ -15,11 +16,13 @@ import (
 )
 
 const (
-	buildScanAPI                        = "api/v2/ci/build"
-	xrayScanBuildNotSelectedForIndexing = "is not selected for indexing"
-	XrayScanBuildNoFailBuildPolicy      = "No Xray “Fail build in case of a violation” policy rule has been defined on this build"
-	projectKeyQueryParam                = "projectKey="
-	includeVulnerabilitiesQueryParam    = "include_vulnerabilities="
+	BuildScanAPI                          = "api/v2/ci/build"
+	xrayScanBuildNotSelectedForIndexing   = "is not selected for indexing"
+	XrayScanBuildNoFailBuildPolicy        = "No Xray “Fail build in case of a violation” policy rule has been defined on this build"
+	projectKeyQueryParam                  = "projectKey="
+	includeVulnerabilitiesQueryParam      = "include_vulnerabilities="
+	buildScanResultsPostApiMinXrayVersion = "3.75.13"
```

Comment:
This version is not final yet. It needs to be validated and updated here and in the prepareGetResultsRequest function comment below.